<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckStatus</class>
 <widget class="QDialog" name="CheckStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="bioProgressBar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="nonStatusButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Nonbiodegradable</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="nonProgressBar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="recyStatusButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Recyclable</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="recyProgressBar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="bioStatusButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Biodegradable</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
